packages/client/src/components/TransactionList.js

```diff
@@ -1,78 +1,47 @@
 import React from "react";
-import { useModalContext } from "../contexts";
-import { getFlowscanUrlForTransaction, shortenAddr } from "../utils";
+import { useModalContext } from "contexts";
 import TransactionStatusIcon from "./TransactionStatusIcon";
 
-const FALLBACK_RECIPIENT_NAME = "No name";
+const maxWidths = {
+  index: 50,
+  status: 140,
+  date: 160,
+  action: 70,
+};
 
-function getDisplayStringForTokenTransfer(tokenTransfer) {
-  if (!tokenTransfer) {
-    return "";
-  }
+const Column = ({ className = "", children, style = {} }) => (
+  <div 
+    className={`column is-flex is-align-items-center ${className}`} 
+    style={style}>
+    {children}
+  </div>
+);
 
-  const amount = parseFloat(tokenTransfer.amount.value ?? "0");
-  const token = tokenTransfer?.amount?.token?.id
-    .split(".")[2]
-    .replace("Token", "")
-    .toUpperCase();
-
-  return `${amount} ${token}`;
-}
-
-function TransactionDetails({ safeOwners, transaction, onClose }) {
-  const renderConfirmation = (address) => {
-    return (
-      <div
-        key={address}
-        className="has-background-black rounded-sm has-text-white is-flex is-align-items-center p-4 mr-2"
-      >
-        {shortenAddr(address)}
-      </div>
-    );
+function getUserFriendlyIntent(intent) {
+  const friendlyNameMap = {
+    "FlowToken": "FLOW",
   };
+  const newWords = intent.split(" ").map(word => {
```

Comment:
Can we move this to the utils?

Reply:
I think utils should be reserved for functions that are used in several places

Reply:
The function is to display the intent in a more friendly way, I can see this could be used in other places.